WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java

```diff
@@ -425,7 +425,8 @@ protected Boolean doInBackground(Void... params) {
                 return false;
             }
 
-            final int maxImageWidth = ImageUtils.getMaximumImageWidthForComment(mContext);
+            final int maxImageWidth = (int) mContext.getResources().getDimension(R.dimen.reader_comment_embedded_image_max_size); //ImageUtils.getMaximumImageWidthForComment(mContext);
```

Comment:
agh! that's ugly - ok makes sense, let me work on that and get back to you. Thanks!